Sources/RabbitMq/Consumer/ConsumerChannel.swift

```diff
@@ -0,0 +1,19 @@
+import AsyncAlgorithms
+
+public final class ConsumerChannel<Element: Sendable>: Sendable {
```

Comment:
It would be nice to conform this to `AsyncSequence`.

Reply:
If you could help me with this particular thing I'd appreciate it! You have a much better idea of how to do this than me.